Sources/MetriportSDK/MetriportClient.swift

```diff
@@ -95,6 +74,14 @@ public class MetriportClient {
 //                          print(workout.startDate)
 //                          print(workout.duration)
 //                          print(workout.workoutActivityType.rawValue)
+//
+//                          if #available(iOS 16.0, *) {
+//                              print(workout.allStatistics.values)
+//                              print(workout.statistics(for: HKObjectType.quantityType(forIdentifier: .activeEnergyBurned)!))
+//                              print(workout.statistics(for: HKObjectType.quantityType(forIdentifier: .distanceWalkingRunning)!)?.sources)
+//
+////                              print(workout.)
+//                          }
 //                          print("")
```

Comment:
The workout would not return sources data such as calories and distance. Will test with another device to see if its because of manually entering data